EmployeeList/Models/Employee.swift

```diff
@@ -0,0 +1,42 @@
+//
+//  Employee.swift
+//  EmployeeList
+//
+//  Created by Wang Siman on 1/29/21.
+//  Copyright © 2021 None. All rights reserved.
+//
+
+import Foundation
+import UIKit
+
+enum EmployeeType: String, Decodable {
+    case FULL_TIME
+    case PART_TIME
+    case CONTRACTOR
+}
+
+struct Employee: Decodable, Identifiable {
+    var id: UUID
+    var fullName: String
+    var phoneNumber: String?
+    var emailAddress: String
+    var biography: String?
+    var photoUrlSmall: String?
```

Comment:
If this is a URL, why isn't it a URL, instead of a String?

Reply:
There is no specific reason. To be honset, I don't know using URL type is a convention.

Reply:
URLs are Codable, and make more sense semantically.